From 8c722c8eaf6972ac21b44df7decb24c7059cf91c Mon Sep 17 00:00:00 2001
From: adra0 <36550102+adra0@users.noreply.github.com>
Date: Fri, 16 Feb 2018 22:45:18 +0200
Subject: [PATCH] Fix i686 build

Otherwise build fails with the following error:
"
CMake Error at CMakeLists.txt:152 (message):
Unsupported target architecture 'i686'.  No support_???.cpp file exists for
this architecture.
"
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 41772295..8b915dc7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,7 +28,7 @@ include (OptimizeForArchitecture)
 
 vc_determine_compiler()
 
-if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(x86|AMD64|amd64)")
+if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(i686|x86|AMD64|amd64)")
    set(Vc_X86 TRUE)
 elseif("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(arm|aarch32|aarch64)")
    message(WARNING "No optimized implementation of the Vc types available for ${CMAKE_SYSTEM_PROCESSOR}")
-- 
2.19.1

